crash when disable WITH_SWIG_PY

My command is 

> cmake .. -DWITH_GPU=OFF -DWITH_SWIG_PY=OFF -DWITH_STYLE_CHECK=OFF -DWITH_TESTING=OFF  -DWITH_DSO=OFF
make all -j `nproc`
make install

Then got 

> make[2]: *** No rule to make target `../python/python_api_wheel', needed by `python/CMakeFiles/paddle_python'.  Stop.
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [python/CMakeFiles/paddle_python.dir/all] Error 2
make: *** [all] Error 2

Maybe forgot some conditions in `python/CMakeFiles.txt`?

--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -50,8 +50,11 @@ add_custom_command(OUTPUT ${PADDLE_PYTHON_BUILD_DIR}/.timestamp
     COMMAND ${CMAKE_COMMAND} -E copy_directory ${PADDLE_PYTHON_BUILD_DIR}/lib* ${PADDLE_PYTHON_BUILD_DIR}/lib-python
     DEPENDS gen_proto_py copy_paddle_pybind framework_py_proto ${PY_FILES} ${external_project_dependencies} ${COPY_PADDLE_MASTER})
 
-add_custom_target(paddle_python ALL DEPENDS
-    ${PADDLE_PYTHON_BUILD_DIR}/.timestamp paddle_pserver_main paddle_trainer paddle_merge_model python_api_wheel ${MKL_DEPENDS})
+set(paddle_python_deps ${PADDLE_PYTHON_BUILD_DIR}/.timestamp paddle_pserver_main paddle_trainer paddle_merge_model ${MKL_DEPENDS})
+if(WITH_SWIG_PY)
+    list(APPEND paddle_python_deps python_api_wheel)
+endif()
+add_custom_target(paddle_python ALL DEPENDS ${paddle_python_deps})
 
 set(PADDLE_PYTHON_PACKAGE_DIR ${CMAKE_CURRENT_BINARY_DIR}/dist/)
 

